src/services/users/reset-password/reset-password.test.js

```diff
@@ -0,0 +1,45 @@
+const { expect } = require('chai')
```

Comment:
might have deleted my comment mistakenly. I'm sure it would be in your email notifications. left a review about the file name reflecting the code file

so if the file name is `index.js`, test file would be `index.test.js`